src/MagnumPlugins/AssimpImporter/AssimpImporter.cpp

```diff
@@ -786,10 +786,19 @@ Containers::Pointer<AbstractMaterialData> AssimpImporter::doMaterial(const Unsig
        something else, so we just don't import it at all. */
     aiColor3D ambientColor;
     UnsignedInt ambientTexture{};
+    UnsignedInt ambientCoordinateSet{};
     mat->Get(AI_MATKEY_COLOR_AMBIENT, ambientColor);
-    if(flags & PhongMaterialData::Flag::AmbientTexture)
+    if(flags & PhongMaterialData::Flag::AmbientTexture) {
         ambientTexture = firstTextureIndex++;
-    else {
+        mat->Get(AI_MATKEY_UVWSRC(aiTextureType_AMBIENT, 0), ambientCoordinateSet);
+
+        if(!configuration().value<bool>("allowMaterialTextureCoordinateSets") && ambientCoordinateSet != 0) {
+            Error{} << "Trade::AssimpImporter::material(): multiple texture coordinate sets are not supported";
+            return {};
+        } else if(ambientCoordinateSet != 0) {
+            flags |= PhongMaterialData::Flag::TextureCoordinateSets;
+        }
```

Comment:
Actually, why not

```suggestion
        if(ambientCoordinateSet != 0) {
            if(!configuration().value<bool>("allowMaterialTextureCoordinateSets")) {
                Error{} << "Trade::AssimpImporter::material():  multiple texture coordinate sets are not allowed by default, enable allowMaterialTextureCoordinateSets to import them";
                return {};
            }
            
            flags |= PhongMaterialData::Flag::TextureCoordinateSets;
        }
```

? Seems more straightforward to me. Same elsewhere